add mtls header base configuring doc (#1298)
wso2/docs-apim: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md

> -   [Create an API Secured with Mutual SSL](#SecuringAPIswithMutualSSL-CreateanAPISecuredwithMutualSSL)

Reviewer: ```suggestion
-   [Create an API Secured with Mutual SSL](#invoke-an-api-secured-with-mutual-ssl-using-postman)
```

> -   [Invoke an API Secured with Mutual SSL from the Developer Portal](#SecuringAPIswithMutualSSL-InvokeanAPIsecuredwithMutualSSLfromtheDeveloperPortal)

Reviewer: ```suggestion
-   [Invoke an API Secured with Mutual SSL from Postman](#invoke-an-api-secured-with-mutual-ssl-using-postman)
```

-   Resource level security will not be applicable for the APIs that are only protected with mutual SSL.

-   Scope level security will not be applicable for the APIs that are only protected with mutual SSL.

> ### Invoke an API secured with Mutual SSL though SSL Terminated Load Balancer.

Reviewer: ```suggestion
### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```

Reviewer: Also fix the link

> When SSL Termination of invocation request happens at the loadbalancer, following prerequisites need to be met from the load balancer.

Reviewer: ```suggestion
When SSL Termination of invocation request happens at the loadbalancer/reverse proxy, the following prerequisites need to be met from the load balancer.
```

-   Terminate the mutual SSL connection from client

> -   Offload the Terminated SSL certficate from client into the Gateway in a Header.(eg: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate)

Reviewer: ```suggestion
-   Pass the client SSL certificate to the Gateway in an HTTP Header. ( eg: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate )
```

Below diagram show how MutualSSL works in such environment.

![]({{base_path}}/assets/img/learn/mtls-loadbalancer.png)

> Default WSO2 API Manager supports to retrieve Certificate from **X-WSO2-CLIENT-CERTIFICATE** header. In order to change the retrival header,
> follow the below instructions.

Reviewer: ```suggestion
By default, the WSO2 API Manager retrieves the client certificate from the **X-WSO2-CLIENT-CERTIFICATE** HTTP header. In order to change the header,
```

-   Resource level security will not be applicable for the APIs that are only protected with mutual SSL.

-   Scope level security will not be applicable for the APIs that are only protected with mutual SSL.

> ### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.

Reviewer: need a new line above this i think.

> ### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.

Reviewer: ```suggestion
### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```
```suggestion

### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```

> When SSL Termination of invocation request happens at the loadbalancer/reverse proxy,the following prerequisites need to be met from the load balancer.

Reviewer: ```suggestion
When SSL Termination of API requests happens at the loadbalancer/reverse proxy, the following prerequisites need to be met from the load balancer.
```

-   Terminate the mutual SSL connection from client
-   Pass the client SSL certificate to the Gateway in an HTTP Header. ( eg: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate )

> Below diagram show how MutualSSL works in such environment.

Reviewer: ```suggestion
The below diagram shows how MutualSSL works in such environment.
```

![]({{base_path}}/assets/img/learn/mtls-loadbalancer.png)

By default, the WSO2 API Manager retrieves the client certificate from the **X-WSO2-CLIENT-CERTIFICATE** HTTP header. In order to change the header,

-  Navigate to the `<API-M_HOME>/repository/conf/deployment.toml` file.

> -  Configure the *certificate_header* under the [apimgt.mutual_ssl]. 

Reviewer: ```suggestion
-  Configure the *certificate_header* under the [apimgt.mutual_ssl] configuration. 
```
